pkgs/data/themes/catppuccin-sddm/default.nix

```
@@ -0,0 +1,72 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchFromGitHub,
+  qtgraphicaleffects,
+  qtsvg,
+  qtquickcontrols2,
+  flavor ? "mocha",
+  font ? "Noto Sans",
+  fontSize ? "9",
+  background ? null,
+  loginBackground ? false,
+}:
+stdenvNoCC.mkDerivation {
+  pname = "catppuccin-sddm";
+  version = "main-2024-03-17";
+
+  src = fetchFromGitHub {
+    owner = "catppuccin";
+    repo = "sddm";
+    rev = "808b4b65c044cd13287d5956ccf2bf49a7d2615f";
+    hash = "sha256-ddz++Xzh3tiEdEnYvIhKcN8Y3FRSvaN0wtmCMb7DEVc=";
+  };
+
+  dontWrapQtApps = true;
+
+  propagatedBuildInputs = [
+    qtgraphicaleffects
+    qtsvg
+    qtquickcontrols2
+  ];
+
+  installPhase = ''
+    mkdir -p "$out/share/sddm/themes/"
+    cp -r src/catppuccin-${flavor} "$out/share/sddm/themes/catppuccin-${flavor}"
+
+    configFile=$out/share/sddm/themes/catppuccin-${flavor}/theme.conf
+
+    substituteInPlace $configFile \
+        --replace 'Font="Noto Sans"' 'Font="${font}"'
+
+    substituteInPlace $configFile \
```

```suggestion
        --replace-fail 'Font="Noto Sans"' 'Font="${font}"' \
```

Please use the new --replace-fail argument everywhere